process/smartContract/hooks/blockChainHook.go

```diff
@@ -596,9 +600,7 @@ func (bh *BlockChainHookImpl) SetCurrentHeader(hdr data.HeaderHandler) {
 func (bh *BlockChainHookImpl) SaveCompiledCode(codeHash []byte, code []byte) {
 	bh.compiledScPool.Put(codeHash, code, len(code))
 	err := bh.compiledScStorage.Put(codeHash, code)
-	if err != nil {
-		log.Debug("SaveCompiledCode", "error", err, "codeHash", codeHash)
-	}
+	log.LogIfError(err, "codeHash", codeHash)
```

Comment:
`log.LogIfError(err, "func", "BlockChainHookImpl.SaveCompiledCode: compiledScStorage.Put")`

Reply:
Right. Missed it. Added it